lib/ipmi_backend_utils.pm

```diff
@@ -145,8 +146,27 @@ sub umount_installation_disk {
     assert_script_run("ls $mount_point");
 }
 
+#Get the partition where the new installed system is installed to
 sub get_installation_partition {
-    my $partition = script_output("fdisk -l | grep \"^\/dev\/sda\.\*\\\*\" | cut -d ' ' -f 1");
+    my $partition = '';
+
+    #Do not use script_output because when the command fail, script_output dies
+    type_string(qq{fdisk -l | grep "^/dev/sda.*\\*" | cut -d ' ' -f 1 | tee /dev/$serialdev\n});
+    $partition = wait_serial;
+    save_screenshot;
+    if (is_storage_ng && ($partition eq '')) {
+        record_soft_failure "bsc#1080729 - Partitioner does not mark boot flag";
```

Comment:
this bug is now marked as duplicate of https://bugzilla.suse.com/show_bug.cgi?id=1076424 which is marked as VERIFIED FIXED so please make sure to remove this code section again

Reply:
Just confirm the common rule:

You can see here the solution is provided as a further check which will be executed only when the normal way fail, instead of a direct replacement. So even if the bug is verified fixed,  current code change still works, and can help when regression happens.

Then shall it still be removed?

Reply:
Well, a soft-fail should only be shown for bugs which are still open. I recommend to either remove the record_soft_fail or replace it by a record_info

Reply:
okurz, thanks for reply. The code logic is when the fdisk way does not work, then it record_soft_fail, not always record_soft_fail. And when that way does not work, it does show issue happen again, and the bug should be reopened, so record_soft_fail rules meet again. I agree that the bug number I give is DUPLICATE and should be corrected to the non-dup one. But I still do not understand why record_soft_fail should be removed.